### ISSUE TITLE
There is no way to regenerate pretty urls when a parent section has been moved

### ISSUE THREAD

**Issue by bryanpizzillo** [NCIOCPL/cgov-digital-platform#2784]
### Issue description

**TL;DR; Make /admin/config/search/path/update_bulk available to a Site Admin.**

The ideal scenario is that the URL aliases of all the nodes and media under that section should be automatically updated when the parent changes. Instead only the child terms for that section are updated. However, there are issues with pathauto updating in bulk upon site section term save.

The alternate is to expose the `/admin/config/search/path/update_bulk` route (and only that route from pathauto) to the Site Admin role so that anyone can regenerate path aliases.

This action should ONLY be done right before a cache clearing since it does not seem to invalidate any Drupal cache tags.

Right now IAs are repushing every page through the workflow under a section that changed. This could be hundreds of pages. That is unacceptable. (The workflow at least invalidates the cache) 

> **ESTIMATE** NN

### What's the expected change?
- I should not get a "You are not authorized to access this page." message.

### What's the current functionality?
- I get a "You are not authorized to access this page." message.
